<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLDayLightDlg</class>
 <widget class="QDialog" name="FLDayLightDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Day light saving setting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Daily clock adjustment(Secs):</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>-15</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Enable Daylight saving</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>On:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>1st</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2nd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3rd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4th</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5th</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Monday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tuesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wednesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thursday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Friday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunday</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QComboBox" name="comboBox_5">
        <item>
         <property name="text">
          <string>January</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>February</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>March</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>April</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>May</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>June</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>July</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>August</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>September</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>October</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>November</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>December</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit_2"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>On:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>in</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>1st</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2nd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3rd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4th</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5th</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBox_4">
        <item>
         <property name="text">
          <string>Monday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tuesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wednesday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thursday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Friday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Saturday</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sunday</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="comboBox_6">
        <item>
         <property name="text">
          <string>January</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>February</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>March</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>April</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>May</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>June</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>July</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>August</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>September</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>October</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>November</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>December</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
